src/app/component/CmsBlock/CmsBlock.component.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+import Html from 'Component/Html';
+import { BlockListType } from 'Type/CMS';
+import { ChildrenType } from 'Type/Common';
+
+/**
+ * Cms Block
+ * @class CmsBlock
+ */
+class CmsBlock extends PureComponent {
+    render() {
+        const { identifiers, blocks: { items = {} }, children } = this.props;
+
+        const cmsBlocks = identifiers.reduce((acc, id) => (items[id]
```

Comment:
Please collect the array of cms block content in container. Then, in component render it with `Html`.